apps/dotcms-ui/src/app/portlets/dot-templates/dot-template-list/dot-template-list.component.spec.ts

```diff
@@ -448,6 +470,28 @@ describe('DotTemplateListComponent', () => {
 
                 expect(publishTemplate.actions).toEqual(actions);
             });
+
+            describe('template as a file ', () => {
+                it('should go to site Broser when selected', () => {
+                    spyOn(dotSiteBrowserService, 'setSelectedFolder').and.returnValue(of({}));
+                    const rows: DebugElement[] = fixture.debugElement.queryAll(
+                        By.css('.p-selectable-row')
+                    );
+                    fixture.detectChanges();
+                    rows[rows.length - 1].nativeElement.click();
```

Comment:
triggerEvent gives error, because primeNg component checks some properties of the real event.